--- uae-0.8.16/configure.in~	Mon Feb 19 22:59:48 2001
+++ uae-0.8.16/configure.in	Thu Sep 20 06:10:58 2001
@@ -31,9 +31,9 @@
 ], [uae_cv_prog_cc_watcom=yes], [uae_cv_prog_cc_watcom=no])
 AC_MSG_RESULT($uae_cv_prog_cc_watcom)
 
-AM_CYGWIN32
-AM_MINGW32
-AM_EXEEXT
+AC_CYGWIN32
+AC_MINGW32
+AC_EXEEXT
 
 NR_WARNINGS=0
 NR_ERRORS=0
@@ -45,11 +45,6 @@
 
 NEED_THREAD_SUPPORT=no
 
-ADDITIONAL_CFLAGS=
-OPTIMIZE_CFLAGS=
-WARNING_CFLAGS=
-DEBUG_CFLAGS=-g
-NO_SCHED_CFLAGS=
 LDFLAGS="$LDFLAGS -L."
 
 dnl Checks for libraries.
@@ -536,26 +531,7 @@
 ], [AC_MSG_RESULT(yes); HAVEGLIBC2=yes], AC_MSG_RESULT(no))
 fi
 
-if test $ac_cv_prog_gcc = yes; then
-  WARNING_CFLAGS="-Wall -Wno-unused -Wno-format -W -Wmissing-prototypes -Wstrict-prototypes"
-  OPTIMIZE_CFLAGS="-O2 -fomit-frame-pointer"
-  dnl Not nice, but needed to turn off -g
-  CFLAGS=
-fi
-
-if [[ "x$DO_PROFILING" = "xyes" ]]; then
-  if [[ "x$CC" = "xgcc" ]]; then
-    OPTIMIZE_CFLAGS="-O"
-    DEBUG_CFLAGS="-g -fno-inline -fno-omit-frame-pointer -pg -DUSE_PROFILING"
-    LDFLAGS="-pg"
-  else
-    DO_PROFILING=no
-    echo "Don't know how to set up profiling for your compiler."
-    NR_ERRORS=`expr $NR_ERRORS + 1`
-  fi
-fi
-
-CFLAGS="$OPTIMIZE_CFLAGS $DEBUG_CFLAGS $WARNING_CFLAGS $CFLAGS $ADDITIONAL_CFLAGS"
+CFLAGS="$CFLAGS $CFLAGS $ADDITIONAL_CFLAGS"
 
 if [[ "x$HAVEGCC27" = "xyes" ]]; then
   CFLAGS="$CFLAGS -DGCCCONSTFUNC=\"__attribute__((const))\""
