--- uae-0.8.25/src/main.c	2006-06-07 17:41:49.000000000 +0200
+++ uae-0.8.25_patched/src/main.c	2006-06-08 20:33:05.000000000 +0200
--- uae-0.8.29/src/main.c~	2009-11-08 18:22:11.264214266 +0100
+++ uae-0.8.29/src/main.c	2009-11-08 18:22:38.697407599 +0100
@@ -506,8 +506,8 @@
 	 * don't use the optionsfile variable, so that we will save changes
 	 * to the home directory.  */
 	char pwd_optionsfile[256];
-	strcpy (optionsfile, OPTIONSFILENAME);
-	cfgfile_load (&currprefs, optionsfile);
+	strcpy (pwd_optionsfile, OPTIONSFILENAME);
+	cfgfile_load (&currprefs, pwd_optionsfile);
 #endif
     }
     printf("optsfile: %s\n", optionsfile );
