diff -ur uae-0.8.25.orig/src/sd-file/sound.c uae-0.8.25/src/sd-file/sound.c
--- uae-0.8.25.orig/src/sd-file/sound.c	2005-07-01 17:53:26.000000000 +0200
+++ uae-0.8.25/src/sd-file/sound.c	2006-01-12 18:56:01.011331250 +0100
@@ -94,7 +94,7 @@
     buf[t+2] = (v>>16) & 255;
     buf[t+3] = (v>>24) & 255;
 
-    t = 20; v = 0x00010001 + (currprefs.stereo ? 0x10000 : 0);
+    t = 20; v = 0x00010001 + (currprefs.sound_stereo ? 0x10000 : 0);
     buf[t] = v & 255;
     buf[t+1] = (v>>8) & 255;
     buf[t+2] = (v>>16) & 255;
@@ -106,13 +106,13 @@
     buf[t+2] = (v>>16) & 255;
     buf[t+3] = (v>>24) & 255;
     t = 32; v = ((currprefs.sound_bits == 8 ? 1 : 2)
-		 * (currprefs.stereo ? 2 : 1)) + 65536*currprefs.sound_bits;
+		 * (currprefs.sound_stereo ? 2 : 1)) + 65536*currprefs.sound_bits;
     buf[t] = v & 255;
     buf[t+1] = (v>>8) & 255;
     buf[t+2] = (v>>16) & 255;
     buf[t+3] = (v>>24) & 255;
     t = 28; v = (currprefs.sound_freq * (currprefs.sound_bits == 8 ? 1 : 2)
-		 * (currprefs.stereo ? 2 : 1));
+		 * (currprefs.sound_stereo ? 2 : 1));
     buf[t] = v & 255;
     buf[t+1] = (v>>8) & 255;
     buf[t+2] = (v>>16) & 255;
@@ -123,10 +123,10 @@
 
     if (currprefs.sound_bits == 16) {
 	init_sound_table16 ();
-	sample_handler = currprefs.stereo ? sample16s_handler : sample16_handler;
+	sample_handler = currprefs.sound_stereo ? sample16s_handler : sample16_handler;
     } else {
 	init_sound_table8 ();
-	sample_handler = currprefs.stereo ? sample8s_handler : sample8_handler;
+	sample_handler = currprefs.sound_stereo ? sample8s_handler : sample8_handler;
     }
     sound_available = 1;
     sndbufsize = 44100;
